mender-convert

```diff
@@ -44,6 +44,12 @@ function show_version() {
 }
 
 function trap_exit() {
+  EXIT_CODE=$?
+  if [[ ${EXIT_CODE} -ne 0 && ${EXIT_CODE} -ne ${FATAL_EXIT_CODE} ]]; then
+    log_error "mender-convert failed"
+    tac work/convert.log | sed '/DEBUG/q' | tac | sed 's/Running/When running/'
+    log_error "mender-convert failed due to this commands' exit code: ${EXIT_CODE}"
```

Comment:
We already said which command failed in line above, I think this can be reduced to:

```
log_error "mender-convert exit code: ${EXIT_CODE}"
```